pkg/client/clientset/versioned/scheme/register.go

```diff
@@ -36,6 +38,8 @@ var localSchemeBuilder = runtime.SchemeBuilder{
 	kyvernov1.AddToScheme,
 	kyvernov1alpha1.AddToScheme,
 	policyv1alpha1.AddToScheme,
+	appsv1.AddToScheme,
+	corev1.AddToScheme,
```

Comment:
Why do we need to register these 2 groups? What about other k8s built-in groups?

Reply:
while applying deployment manifest as resource I was getting the following error:
```Error: failed to load resources 
Cause: no kind "Deployment" is registered for version "apps/v1" in scheme "pkg/runtime/scheme.go:101"
```
To resolve the above issue I added `appsv1.AddToScheme`.

While applying pod manifest as resource I was getting similar error. To resolve this, I added `corev1.AddToScheme`.